npm/src/commands/install.js

```diff
@@ -1,76 +1,90 @@
 // @flow
-import mkdir from 'mkdirp'
-import {getGHLibsAndFlowVersions, filterDefs, formatDefTable}
-  from "../lib/libDef.js";
-import type {LibDefWithFlow} from "../lib/libDef.js";
+
+import mkdirp from 'mkdirp'
+import {
+  getGHLibsAndFlowVersions,
+  filterDefs,
+  formatDefTable
+} from '../lib/libDef';
+
+import type {LibDefWithFlow} from '../lib/libDef';
 import {child_process, path, https, fs} from '../lib/node'
 
-export const name = "install";
-export const description =
-  "Installs a libdef";
+export const name = 'install';
+export const description = 'Installs a libdef to the ./flow-typed directory';
+
+export function setup(yargs: Object): Object {
+  return yargs
+    .usage(`$0 ${name} - ${description}`)
+    .options({
+      flowVersion: {
+        demand: true,
+        describe: 'The version of Flow fetched libdefs must be compatible with',
+        type: 'string',
+      },
+    });
+};
+
+function failWithMessage(message: ?string) {
+  if (message != null) {
+    console.error(message);
+  }
+  return 1;
+}
 
 type Args = {
   _: Array<string>;
   flowVersion?: string;
 }
-export async function run(args: Args): Promise<number> {
-
-  function errorNeedDefName() {
-    console.error('Please provide a libdef name ' +
-      '(example: lodash, or lodash@4.2.1)')
-  }
 
-  function errorNeedFlowVersion() {
-    console.error('Please provide a flow version ' +
-    '(example: --flow v0.23.0)')
-  }
+export async function run(args: Args): Promise<number> {
+  const { _ } = args;
 
-  if (!args._ || !(args._.length > 1)) {
-    errorNeedDefName()
-    return 1;
+  if (_ == null || !(_.length > 1)) {
+    return failWithMessage('Please provide a libdef name (example: lodash, or lodash@4.2.1)');
   }
 
-  if (!args.flow) {
-    errorNeedFlowVersion()
-    return 1;
+  if (args.flowVersion == null) {
+    return failWithMessage('Please provide a flow version (example: --flow 0.23)');
```

Comment:
If the arg is changed to `flowVersion` as on line, 53, the help string should be `--flowVersion 0.23`